src/oasis-nexus/api.ts

```diff
@@ -267,12 +281,18 @@ export const useGetConsensusAccountsAddress: typeof generated.useGetConsensusAcc
   })
 }
 
-export const useGetRuntimeAccountsAddress: typeof generated.useGetRuntimeAccountsAddress = (
-  network,
-  runtime,
-  address,
-  options?,
-) => {
+// TODO: remove conversion when API supports querying by EVM address
+type useGetRuntimeAccountsAddressExtended = (
+  searchParam: string | undefined,
+  ...rest: Parameters<typeof generated.useGetRuntimeAccountsAddress>
+) => UseQueryResult<
+  Awaited<ReturnType<typeof GetRuntimeAccountsAddress>>,
+  HumanReadableErrorResponse | NotFoundErrorResponse
+> & { queryKey: QueryKey }
+
+export const useGetRuntimeAccountsAddress: useGetRuntimeAccountsAddressExtended = (searchParam, ...rest) => {
```

Comment:
I think it would be much prettier if we can call useGetRuntimeAccountsAddress with address=oasis1|0x, and move address loader logic into this wrapper. Especially if API will behave like that in the future

Reply:
I am not sure, based on this [TODO](https://github.com/oasisprotocol/explorer/blob/40bf1517148c0a60c82dd92207a66c6fa3d32f0b/src/app/utils/route-utils.ts#L139)(Written by you) I would expect that the API will support 0x addresses in the future.

But I agree, it would be much cleaner if I move this to the API wrapper call. It does affect the token address call as well. Will look into that.